frontend/src/static/css/style.scss

```diff
@@ -715,3 +723,17 @@ header {
     margin: 0;
   }
 }
+
+/* Tags in commits */
+.tag {
```

Comment:
One last question, shouldn't this tag be inside v_zoom only? Can change the place of this tag?

Reply:
Yes can do. I placed it outside as the `commit-message` class is placed out of v_zoom as well.

Reply:
Tried to move the tag to be within v_zoom. Since the tag is in two different places (on top of the ramp and on each commit), it is better to place the `.tag` css class outside as if we put inside then might need to duplicate in two places.

Reply:
Maybe we should put commit-message inside v_zoom as well? I don't think we need to duplicate the code.

Reply:
Moved the `.tag` class into `.zoom`. Should I move `commit-message` in this PR as well? As this PR maybe unrelated to moving `commit-message`

Reply:
No need to move commit-message.